signalmap.go

```diff
@@ -9,40 +9,76 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+const (
+	sigrtmin = 34
+	sigrtmax = 64
```

Comment:
Go SDK doesn't provide `SIGRTMIN` and `SIGRTMAX`. It could be a good idea to go there first and return here afterwards, but I don't think I have great chances. Docker/Moby didn't do it.

Reply:
You could access them through CGo (though it would be a bit ugly).

Reply:
Yeah, I know. It's how it's done in Go SDK for some OSes (but not for Linux -- for Linux it's hard-coded). It's hard-coded in Python, too. And in Moby/Docker. Different people have different opinions.
But signal map completeness is not the issue. See #1706